cryptofeed/backends/_util.py

```diff
@@ -53,3 +53,27 @@ def book_convert(book: dict, data: dict, depth: int, convert=str):
         count += 1
         if depth and count >= depth:
             break
+
+
+def book_flatten(feed: str, pair: str, book: dict, timestamp: float, delta: str) -> dict:
```

Comment:
it might be worth having 2 functions here, book_flatten_l2 and book_flatten_l3. They can be wrappers for the same shared code, but since L3 has the order_id field and L2 doesn't, it might be worth making them separate.

Reply:
true, but most the code is the same, so its a lot easier to have one extra check in here as opposed to duplicating the majority of the function.